docs/en/observability/logs-troubleshooting.asciidoc

```diff
@@ -1,4 +1,111 @@
 [[logs-troubleshooting]]
 = Troubleshoot your logs
 
-_coming soon_
\ No newline at end of file
+This section provides solutions to errors you might encounter while onboarding your logs in {kib}.
+
+[discrete]
+[[logs-troubleshooting-insufficient-priv]]
+== User does not have permissions to create API key
+
+If you don't have the required privileges to create an API key, you'll see the following error message:
+
+----
+User does not have permissions to create API key.
+
+Required cluster privileges are [`monitor`, `manage_own_api_key`] and 
+required index privileges are [`auto_configure`, `create_doc`] for 
+indices [`logs-*-*`, `metrics-*-*`], please add all required privileges 
+to the role of the authenticated user.
+----
+
+*Solution:*
+
+You need an administrator to give you the necessary permissions listed in the error messages and then to restart the onboarding flow.
```

Comment:
To make this more actionable, could we share here the API call that the admin would have to make? Otherwise the admin comes back to the user and asks what exactly they need to do.

Reply:
@ruflin Same as above. Could someone provide this API call.

Reply:
@weltenwort ^